mobilizon_bots/storage/db.py

```diff
@@ -48,4 +48,8 @@ def gracefully_tear_down():
     logger.info("Shutting down DB")
     loop = asyncio.new_event_loop()
     asyncio.set_event_loop(loop)
-    asyncio.run(Tortoise.close_connections())
+    asyncio.run(tear_down())
```

Comment:
I was wondering if this actually works in case of exception. IIUC this PR is supposed to prevent the program from hanging waiting for Tortoise to close connections on exit. So the problem was that `gracefully_tear_down`, called `atexit`, was called too late i.e. after (or during) the `exit` call in main.

If this is the case I'm not sure this actually works in case of exception since the exception would prevent the call to exit all together.

What about wrapping the contents of `main` in a `try/except` like so:

`exit_code = 0
try:
    [...]
except BaseException as e:
    logger.error(repr(e))
    exit_code = 1
gracefully_exit(exit_code)
`